makiflow/layers/initializers/core/base_uniform.py

```diff
@@ -0,0 +1,31 @@
+# Copyright (C) 2020  Igor Kilbas, Danil Gribanov, Artem Mukhin
+#
+# This file is part of MakiFlow.
+#
+# MakiFlow is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# MakiFlow is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Foobar.  If not, see <https://www.gnu.org/licenses/>.
+
+import numpy as np
+
+from .base_init import BaseInitializer
+
+
+class BaseUniformInitializer(BaseInitializer):
+
+    def _create_matrix(self, shape: list, dtype=np.float32, low=-1.0, high=1.0):
```

Comment:
`low` and `high` arguments are inaccessible to the user here and won't ever change. Move them to the `__init__` method. Then use those fields for random tensor initialization.

```
def __init__(self, low=-1.0, high=1.0):
    super().__init__()
    self._low = low
    self._high = high

def _create_matrix(self, shape, dtype):
    return np.random.uniform(low=self._low, high=self._high, size=shape, dtype=dtype)
```

Reply:
Refactor other uniform initializers in accordance with this change.